base/applications/cmdutils/more/lang/ro-RO.rc

```diff
@@ -1,60 +1,64 @@
 /*
- * FILE:       base/applications/cmdutils/more/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Fulea Ștefan (PM on ReactOS Forum at fulea.stefan)
- * CHANGE LOG: 2011-08-20  initial translation
- *             2011-10-17  diacritics change, other minor changes
+ * PROJECT:     ReactOS More Command
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2019 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE "Afișează date pe ecran, pagină-cu-pagină.\n\
+    IDS_USAGE "Afișarea datelor pe ecran, pagină cu pagină.\n\
 \n\
-MORE [/E [options]] < [drive:][path]filename\n\
-command | MORE [/E [options]]\n\
+MORE [/E [opțiuni]] < [disc:][cale]numele fișierului\n\
+command | MORE [/E [opțiuni]\n\
 \n\
-    [drive:][path]filename  A file whose contents are to be displayed.\n\
-    command                 A command whose output is to be displayed.\n\
+    [disc:][cale]numele fișierului  Un fișier al cărui conținut urmează să fie afișat.\n\
```

Comment:
Two things here. First is that you may want to reconsider the use of the term "disc" (since you agreed to take care of it in other cases). Second, the line is longer than 80 characters, and this is a command line based interface.